File: VB/Controllers/HomeController.vb
Imports Example

Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        ViewData("Message") = "Welcome to DevExpress Extensions for ASP.NET MVC!"

        Dim obj As New DataObject()

        Return View(obj)
    End Function

    Public Function GridViewPartial() As ActionResult
        Dim obj As New DataObject()

        Return PartialView(obj)
    End Function
End Class

File: CS/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace Example.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!";

            DataObject obj = new DataObject();

            return View(obj);
        }

        public ActionResult GridViewPartial() {
            DataObject obj = new DataObject();

            return PartialView(obj);
        }
    }
}
